packages/flutter/lib/src/material/chip.dart

```diff
@@ -1179,6 +1189,20 @@ class _RawChipState extends State<RawChip> with MaterialStateMixin, TickerProvid
     if (!hasDeleteButton) {
       return null;
     }
+    final IconThemeData iconTheme = widget.iconTheme
+      ?? chipTheme.iconTheme
+      ?? theme.chipTheme.iconTheme
+      ?? const IconThemeData(size: _kDeleteIconSize);
```

Comment:
BTW, just wonder whether we can just use the default icon size from `chipDefaults` instead of this constant? Because I'm thinking if the the token is updated to a different value, then this can be updated automatically.

Reply:
Good idea, let me add this.